baselines/teams.md

```diff
@@ -1,15 +1,19 @@
+**`TLP:CLEAR`**
 # CISA M365 Security Configuration Baseline for Teams
 
-Microsoft Teams is a cloud-based text and live chat workspace in Microsoft 365 that supports video calls, chat messaging, screen-sharing, and file sharing.. This Secure Configuration Baseline (SCB) provides specific policies to strengthen Microsoft Teams' security.
+Microsoft Teams is a cloud-based text and live chat workspace in Microsoft 365 (M365) that supports video calls, chat messaging, screen sharing, and file sharing. This secure configuration baseline (SCB) provides specific policies to strengthen Microsoft Teams' security.
```

Comment:
Microsoft 365 (M365) Teams is a cloud-based text and live chat workspace that supports video calls, chat messaging, screen sharing, and file sharing. This secure configuration baseline (SCB) provides specific policies to strengthen Microsoft Teams’ security. 

Changing first sentence to standardized intro across SCBs